Cargo.toml

```diff
@@ -17,8 +21,17 @@ object = { version = "0.29.0", default-features = false, features = ['archive']
 [lib]
 crate-type = ["cdylib"]
 
+[profile.release]
+# Omit any unwinding support.
```

Comment:
It may be worth clarifying that this is already the default for wasm so this doesn't actually do anything, but it may be worth specifying if this ever changes in the future.